src/os/src/detail/osFile.c

```diff
@@ -43,8 +44,10 @@ void taosGetTmpfilePath(const char *fileNamePrefix, char *dstPath) {
     strcat(tmpPath, "-%d-%s");
   }
 
-  char os_rand[8] = {0};
-  taosRandStr(os_rand, tListLen(os_rand) - 1);
+  char os_rand[32] = {0};
+
+  sprintf(os_rand, "%" PRIu64, atomic_add_fetch_64(&seqId, 1));
+
   snprintf(dstPath, PATH_MAX, tmpPath, getpid(), os_rand);
```

Comment:
The length of the 'rand' has increased from 8 to 32. Does the 'dstPath' also need to increase the length?